### PR TITLE
Feature/mr 145/fix validations password

### PR DESCRIPTION
**Pull request by Dannylez**
Fixes in modals, update trainer, aside and others

### CODE REVIEW

File: src/Components/MembersHome/Activities/index.jsx [hunk 1]
```diff
@@ -20,11 +20,74 @@ const Activities = () => {
...
                 />
+                  }
+
+                  }
+                )
+                <img
+                  src="https://st.depositphotos.com/1708346/1404/i/450/depositphotos_14043607-stock-photo-college-gym-49ers-football-team.jpg"
+                  alt="Some activity"
+                  className={styles.image}
+                /> */}
```
Comment (DelfiLT) [inline src/Components/MembersHome/Activities/index.jsx:90]: Remove comments

File: src/Components/Shared/Aside/aside.module.css [hunk 2]
```diff
@@ -1,22 +1,21 @@
 @import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
 
-.sidebar{
-  position: sticky;
-  top: 0;
-  left: 0;
+/* .sidebar{
   height: 100vh;
```
Comment (DelfiLT) [inline src/Components/Shared/Aside/aside.module.css:4]: Remove comments

File: src/Components/Shared/Aside/index.jsx [hunk 3]
```diff
@@ -1,45 +1,86 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
 import styles from './aside.module.css';
-import Button from '../Button';
+/* import Button from '../Button';
 import { useDispatch, useSelector } from 'react-redux';
-import { asideOnThunk, asideOffThunk } from 'redux/aside/thunks';
+import { asideOnThunk, asideOffThunk } from 'redux/aside/thunks'; */
```
Comment (DelfiLT) [inline src/Components/Shared/Aside/index.jsx:6]: Remove comments

File: src/Components/Shared/Aside/index.jsx [hunk 4]
```diff
@@ -1,45 +1,86 @@
...
 
 const Aside = () => {
-  const { isOn } = useSelector((state) => state.aside);
-  const dispatch = useDispatch();
+  const role = sessionStorage.getItem('role');
+  /*   const { isOn } = useSelector((state) => state.aside);
+  const dispatch = useDispatch(); */
 
-  const handleSandwichClick = () => {
+  /*   const handleSandwichClick = () => {
```
Comment (DelfiLT) [inline src/Components/Shared/Aside/index.jsx:13]: X28282

**Review by DelfiLT — APPROVED** [review]

**Review by SantiH97 — APPROVED** [review]

**Review by ukiweskamp — APPROVED** [review]

**Pull request merged**